action/flags.go

```diff
@@ -0,0 +1,166 @@
+package action
+
+// FlagHandler contains all flags from cmd
+type FlagHandler struct {
+	Bench                bool   `json:"bench"`
+	ExpectSize           int64  `json:"expect_size"`
+	Format               string `json:"format"`
+	HumanReadable        bool   `json:"human_readable"`
+	LongFormat           bool   `json:"long_format"`
+	MaximumMemoryContent int64  `json:"maximum_memory_content"`
+	Recursive            bool   `json:"recursive"`
+	Reverse              bool   `json:"reverse"`
+	Zone                 string `json:"zone"`
+}
+
+// checkNil returns a non-nil pointer of FlagHandler for embed-in use in action handler
+func (fh *FlagHandler) checkNil() *FlagHandler {
```

Comment:
`checkNil` looks ugly.

Reply:
> `checkNil` looks ugly.

After removing actions on `FlagHandler`, which would be embedded as normal struct instead of pointer, `checkNil` will be deprecated.